src/create-client.ts

```diff
@@ -122,13 +122,22 @@ export async function createClient(
     return memoryStorage.getItem(storageKeys.accessToken) as string | undefined;
   }
 
-  async function getAccessToken() {
+  async function getAccessToken(): Promise<string> {
     // TODO: should this respect onBeforeAutoRefresh ?
     if (_needsRefresh()) {
-      await refreshSession();
+      try {
+        await refreshSession();
+      } catch (err) {
```

Comment:
Should we be testing whether this is an instance of `RefreshError` or `LoginRequiredError`?

Or if you want to just catch all of it and you don't care about the `err`, we can drop the binding:

```suggestion
      } catch {
```

Reply:
I had that check originally (https://github.com/workos/authkit-js/compare/130e84a50eab4156962f4b8d6356f6f5b60b967c..1e9d0c1fd61ae2bdb228745edf1922a15c638fcf ), but then I pulled it out (I figured any error should probably result in the same exception...). i'm having 2nd thoughts now though. will revert!